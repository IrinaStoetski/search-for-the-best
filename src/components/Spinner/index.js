import React from 'react';
import './style.scss';

const Spinner = () => (
  <div className="sk-chase">
    <div className="sk-chase-dot" />
    <div className="sk-chase-dot" />
    <div className="sk-chase-dot" />
    <div className="sk-chase-dot" />
    <div className="sk-chase-dot" />
    <div className="sk-chase-dot" />
  </div>
);

export default Spinner;
