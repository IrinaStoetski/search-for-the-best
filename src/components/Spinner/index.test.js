/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from './index';

test('renders correctly active button', () => {
  const tree = renderer
    .create(<Spinner />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
