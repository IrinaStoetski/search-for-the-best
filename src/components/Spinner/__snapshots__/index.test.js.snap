// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly active button 1`] = `
<div
  className="sk-chase"
>
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
  <div
    className="sk-chase-dot"
  />
</div>
`;
